backends/xnnpack/test/ops/test_conv_transpose2d.py

```diff
@@ -0,0 +1,430 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Thank you for adding these tests. I assume it was non-trivial to overload conv tests for transpose case?